src/main/java/uk/gov/hmcts/reform/prl/services/ReturnApplicationService.java

```diff
@@ -116,8 +124,10 @@ public String getReturnMessageForTaskList(CaseData caseData) {
             }
         }
 
-        returnMsgStr.append("Resolve these concerns and resend your application."
-                                + "You have been emailed the full details of your application return.");
+        returnMsgStr.append("""
+        Resolve these concerns and resend your application.
```

Comment:
merge 127 & 128 please, unless the line is exceeding 150 character limitation

Reply:
This can't be done. As part of text block the content cannot start on the same line after opening quote.